app.js

```diff
@@ -32,26 +32,40 @@ app.get('/', (req, res) => {
   return repository.getMostRecentNewsletter(req, res);
 });
 
-app.post('/sendCampaign/:id', async (req, res, next) => {
+app.post('/sendMailCampaign/:id', async (req, res, next) => {
   const campaign_id = req.params.id;
-  const agendaId = req.query.agendaId;
-  if (!campaign_id || !agendaId) {
-    throw new Error('No campaign id or agenda id.');
+  if (!campaign_id ) {
+    throw new Error('No campaign id.');
   }
   try {
     console.time('SEND MAILCHIMP CAMPAIGN TIME');
     const sendCampaign = await mailchimp.post({
       path: `/campaigns/${campaign_id}/actions/send`
     });
-    await service.generateXML(agendaId, true);
     console.time('SEND MAILCHIMP CAMPAIGN TIME');
     res.send({ sendCampaign });
   } catch (error) {
     next(error);
   }
 });
 
-app.get('/fetchTestCampaign/:id', async (req, res, next) => {
+app.post('/sendToBelga/:id', async (req, res, next) => {
+  const agendaId = req.params.id;
+  if (!agendaId) {
+    throw new Error('No agenda id.');
+  }
+  try {
+    console.time('SEND BELGA CAMPAIGN TIME');
+    await service.generateXML(agendaId, true);
+    console.time('SEND BELGA CAMPAIGN TIME');
+    res.status(201).end();
+  } catch (error) {
+    next(error);
+  }
+});
+
+
```

Comment:
Door het opsplitsen van Mailchimp en Belga is het nu mogelijk een lege nieuwsbrief (geen nota's) te versturen naar Belga.
Want de controle daarop zat in "createCampaign"
Je kon geen campaign aanmaken in dat geval en zonder campaign kon je ook het process niet starten om de nieuwsbrief (en Belga) te versturen
https://github.com/kanselarij-vlaanderen/newsletter-service/blob/feature/KAS-1953_nieuwsbrief_acties/repository/MailchimpService.js#L58

Reply:
Ik zorg ervoor dat die in de frontend opgenomen wordt